includes/managers/icons.php

```diff
@@ -516,6 +517,29 @@ public function register_admin_tools_settings( Tools $settings ) {
 		] );
 	}
 
+	/**
+	 * Get redirect URL when upgrading font awesome.
+	 *
+	 * @return string
+	 */
+	public function get_upgrade_redirect_url() {
+		// PHPCS - No need to verify nonce, the 'redirect_to' could be only post id, and if it is not the
+		// URL will be empty.
+		$document_id = ! empty( $_GET['redirect_to_document'] ) ? $_GET['redirect_to_document'] : null; // phpcs:ignore
+
+		if ( ! is_numeric( $document_id ) ) {
```

Comment:
Why you don'y using `absint()` for the validation?

Reply:
Done